README.md

```diff
@@ -53,10 +53,9 @@ Here you can find the documentation for different versions:
 - Latest stable version: [GitHub docs](https://github.com/flame-engine/flame/tree/master-v0.x/doc)
 - Latest v1.0.0 version: [GitHub docs](https://github.com/flame-engine/flame/tree/1.0.0-rc8/doc)
 
-The complete documentation can be found [here](doc/README.md).
+The complete documentation can be found [here](README.md).
```

Comment:
Doesn't this just link to itself?

Reply:
yeah, I also lost some braincells this weekend apparently, fixed.